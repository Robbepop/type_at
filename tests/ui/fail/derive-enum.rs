use type_at::TypeAt;

#[derive(TypeAt)]
pub enum Enum {}

fn main() {}
