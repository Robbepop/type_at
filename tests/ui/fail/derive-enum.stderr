error: TypeAt derive only works on struct types.
 --> tests/ui/fail/derive-enum.rs:3:10
  |
3 | #[derive(TypeAt)]
  |          ^^^^^^
  |
  = note: this error originates in the derive macro `TypeAt` (in Nightly builds, run with -Z macro-backtrace for more info)
